sdk/spring/spring-cloud-azure-stream-binder-eventhubs/src/main/java/com/azure/spring/cloud/stream/binder/eventhubs/config/EventHubsBinderConfiguration.java

```diff
@@ -110,8 +108,8 @@ public EventHubsMessageChannelBinder eventHubBinder(EventHubsChannelProvisioner
         binder.setBindingProperties(bindingProperties);
         binder.setNamespaceProperties(namespaceProperties.getIfAvailable());
         checkpointStores.ifAvailable(binder::setCheckpointStore);
-        binder.setProducerFactoryCustomizers(producerFactoryCustomizers.orderedStream().collect(Collectors.toList()));
-        binder.setProcessorFactoryCustomizers(processorFactoryCustomizers.orderedStream().collect(Collectors.toList()));
+        producerFactoryCustomizers.orderedStream().forEach(binder::addProducerFactoryCustomizer);
```

Comment:
How about adding `addProducerFactoryCustomizers` to support add multiple customizers one time?

Reply:
not for now, but maybe in later versions